src/h5web/providers/jupyter/utils.ts

```diff
@@ -124,6 +124,22 @@ export function convertDtype(dtype: string): HDF5Type {
         length: length || 'H5T_VARIABLE',
       };
 
+    // Assuming that only booleans are stored as bytes
+    // https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.bool_
+    case 'b':
```

Comment:
Actually, the check could be stricter as we know that boolean dtype is `|b1`. We could then return directly `booleanType` defined in `mock/metadata-utils.ts`

Reply:
Happy to be stricter here and to return `booleanType`. So `|b1` cannot be anything else than a boolean? Or could it be a integer from 0 to 255 as well?

Reply:
After a few tries, arrays with `|b1` are directly interpreted as booleans even in NumPy... I think we are safe :sweat_smile: 

Do we need to move `booleanType` out of `mock/metadata-utils.ts` ?

Reply:
Hmm good point -- just keep it duplicated it, then. Better to keep all the mocked stuff really separate.